### ISSUE TITLE
Annotated sentences - not able to upload from other source

### ISSUE THREAD

**Issue by uklomp** [Signbank/Global-signbank#1399]
@Jetske 
I tried to upload a video and an .eaf from another source of annotated sentences and Signbank doesn't seem ready for this.
It didn't manage to read the glosses, I think maybe because the gloss tier has a different name compared to the .eafs from  the Corpus NGT files, and then it didn't upload. 

![image](https://github.com/user-attachments/assets/6aa8d478-3433-41d9-ae51-a512b2580cfa)

**Comment by uklomp** [comment]
I'll send you an example of the .eaf files we use with this source

**Comment by Jetske** [comment]
That should fix it. I did come across an error for annotated sentence backup files, because it was using examplesentence instead. And it's once again creating id_randomstring files. Note to self: fix this before merging.

**Comment by uklomp** [comment]
Is there an update here, @Jetske? We're almost ready to start uploading from another source.

**Comment by Jetske** [comment]
Yes this is solved, I'll request to merge it. It still makes backup filed for the .eaf files for some reason but that's not a big problem and a different issue.
